pkg/daemon/ceph/client/mgr_test.go

```diff
@@ -0,0 +1,42 @@
+package client
```

Comment:
missing header

Reply:
@travisn correct me but I thought you had added tests for this already?

Reply:
I think the tests were just enabling the pg_autoscaler module in the integration tests